### PR TITLE
Add expert mode to elas game

### PR DESCRIPTION
**Pull request by allardw**
This PR adds an expert mode to the hidden ELAS game

game_expert

### CODE REVIEW

File: asreview/webapp/src/Components/ElasGame.js [hunk 1]
```diff
@@ -51,17 +81,35 @@ const GameStyle = styled(Box)(({ theme }) => ({
...
+  const toggleMode = () => {
+    setMode((prevMode) => (prevMode === "simple" ? "expert" : "simple"));
+    setShuffledOnce(false);
+  };
+
+  // const handleKeyPress = useCallback((event) => {
+  //   if (event.key === "c") {
+  //     setCheatMode((prevCheatMode) => !prevCheatMode);
+  //   }
+  // }, []);
```
Comment (J535D165) [inline asreview/webapp/src/Components/ElasGame.js:102]: Please remove or activate these lines.

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks for your contribution! Please resolve the comment. Will test and merge soon.

**Comment by J535D165** [comment]
And I like the Elas lollypop :)

File: asreview/webapp/src/Components/ElasGame.js (hunk 1 above)
Comment (Rensvandeschoot) [reply asreview/webapp/src/Components/ElasGame.js]: solved

**Review by J535D165 — CHANGES_REQUESTED** [review]
I run into the following:

- Attempt counter doesn't reset when changing modes. 
- Holding the cheat key results in a flickering dialog. It might be better (and easier to implement) to show the cheat when the C key is down and release the cheat mode when the key is up.

**Comment by Rensvandeschoot** [comment]
Changes:

- improved the logic for the different modes
- added an independent counter for the modes 
- removed scoring from the title and placed the scoring board on top of box
- added reset button
- fixed re-shuffling after switching modes

to do:
- [ ] Holding the cheat key results in a flickering dialog.



game_expert

File: asreview/webapp/src/Components/ElasGame.js [hunk 2]
```diff
@@ -50,77 +86,301 @@ const GameStyle = styled(Box)(({ theme }) => ({
...
-  const [imagesArray, setImagesArray] = useState([]);
-  const [paperSelected, setPaperSelected] = useState([]);
-  const [paperSelectedIds, setPaperSelectedIds] = useState([]);
+  // Define initial state
+  const initialState = {
+    mode: "simple",
+    cheatMode: 'inactive',
+    gameStarted: false,
+    imagesArraySimple: initialShuffle("simple"),
+    imagesArrayExpert: initialShuffle("expert"),
```
Comment (J535D165) [inline asreview/webapp/src/Components/ElasGame.js:117]: It's nice to see you removed these and integrated them into 1. These should get removed.

File: asreview/webapp/src/Components/ElasGame.js [hunk 3]
```diff
@@ -50,77 +86,301 @@ const GameStyle = styled(Box)(({ theme }) => ({
...
+    imagesArray: [],
+    isExpertCompleted: false,
```
Comment (J535D165) [inline asreview/webapp/src/Components/ElasGame.js:119]: I prefer not to have different behavior for different games. `isComplete: false`

**Review by J535D165 — APPROVED** [review]

**Pull request merged**